src/platforms/shared/setup/base-context.setup.ts

```diff
@@ -221,26 +220,9 @@ export class BaseContextSetup implements DiProcess {
 	}
 
 	private setupOutstreamPlayers(): void {
-		const outstreamExperiment = new OutstreamExperiment(this.instantConfig);
-		if (this.instantConfig.get('icExCoPlayer') && outstreamExperiment.isExco()) {
-			context.set('services.exCo.enabled', true);
-			context.set('services.distroScale.enabled', false);
-			return;
-		}
-
-		if (this.instantConfig.get('icAnyclipPlayer') && outstreamExperiment.isAnyclip()) {
+		if (this.instantConfig.get('icAnyclipPlayer')) {
 			context.set('services.anyclip.enabled', true);
```

Comment:
```suggestion
    context.set('services.anyclip.enabled', this.instantConfig.get('icAnyclipPlayer'));
```
+ move to the `setServicesContext()`